src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -314,6 +315,12 @@ class SettingsMenu extends Menu {
             gearButton.setAttribute('aria-expanded', true);
         }
 
+        if (getBreakpoint(this.model.get('containerWidth')) === 0 || getBreakpoint(this.model.get('containerWidth')) === 1) {
```

Comment:
Can reduce the number of calls to model in a couple ways, this one is probably the most performant: 

`if (getBreakPoint(this.model.get('containerWidth')) < 2)`

Reply:
We might want to just store the return value of getBreakpoint above all of the if statements here too.